android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraModule.java

```diff
@@ -751,6 +760,29 @@ public void onPictureTaken(byte[] data, Camera camera) {
         }
     }
 
+    private void writeLocationExifData(File cameraRollFile, ReadableMap options) {
+        if(!options.hasKey("metadata"))
+            return;
+
+        ReadableMap metadata = options.getMap("metadata");
+        if (!metadata.hasKey("location"))
+            return;
+
+        ReadableMap location = options.getMap("location");
+        if(!location.hasKey("coords"))
```

Comment:
The RN geo location returns a gps position that looks like this:

```
{
  coords: {lat..., lon...},
  timestamp: 1234454
}
```

I'm not sure what you mean about passing a boolean instead of a gps location.

This change is just bringing the android API in line with the iOS one.